src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -313,6 +313,9 @@ template <typename T>
 T pop_or_default(ov::AnyMap& config, const std::string& key, const T& default_value) {
     auto anyopt = pop_option(config, key);
     if (anyopt.has_value()) {
+        if (anyopt.value().empty()) {
+            return T{};
```

Comment:
Why not `default_value`?
Can you explain, why it is needed now

I guess that's because you switched from exceptions to `return ov::Any();`, but why?

Reply:
Almost any property of type `list`, `set`, or `dict` can potentially be empty and we should be able to accept such values. So Ilya suggested to return `ov::Any()` in bindings and process empty `ov::Any()` in utils. 

Regarding usage of `default_value`, I tried it locally but tests, that use confing shown below failed, because it is not equivalent to creation of empty container. So I changed to `T{}`.
https://github.com/openvinotoolkit/openvino.genai/blob/6bb564474f4b4582f82c811ab0947a82577af5ff/tests/python_tests/test_llm_pipeline_static.py#L19-L20

Reply:
> but why?

yes, we need somehow bypass information from python that container (list, map, set) is empty, but we don't know type of this container (e.g. vector<of what type?>), while in this function we know this type `T`

maybe it will safer to `return T{};` only for containers? because for regular type empty `ov::Any` is still not expected

Reply:
Done.